dspace/solr/search/conf/schema.xml

```diff
@@ -259,6 +259,8 @@
     <field name="taskfor" type="string" indexed="true" stored="true" omitNorms="true" multiValued="true" docValues="true" />
    
     <field name="has_content_in_original_bundle" type="string" indexed="true" stored="true" omitNorms="true" multiValued="true" docValues="true" />
+
+    <field name="submit" type="string" indexed="true" stored="true" omitNorms="true" multiValued="true" docValues="true" />
```

Comment:
We may want to add a comment here to explain the difference between "submit" and the existing "submitter" index (as it may be confusing in the future).  Obviously this index is to track *who has submit permissions*, while "submitter" seems to be *who submitted an item*.  They are similarly named, so we should add a comment here to describe them.